client/login/magic-login/request-login-email-form.jsx

```diff
@@ -10,29 +10,50 @@ import { connect } from 'react-redux';
 import {
 	fetchMagicLoginRequestEmail,
 	hideMagicLoginRequestNotice,
-	setMagicLoginInputEmailAddress,
 } from 'state/login/magic-login/actions';
 import {
-	getMagicLoginEmailAddressFormInput,
-	getMagicLoginEmailAddressFormInputIsValid,
+	recordPageView,
+} from 'state/analytics/actions';
+import {
 	isFetchingMagicLoginEmail,
+	getMagicLoginCurrentView,
 	getMagicLoginRequestEmailError,
 	getMagicLoginRequestedEmailSuccessfully,
 } from 'state/selectors';
+import {
+	CHECK_YOUR_EMAIL_PAGE,
+} from 'state/login/magic-login/constants';
 
+import EmailedLoginLinkSuccessfully from './emailed-login-link-successfully';
 import FormButton from 'components/forms/form-button';
 import FormFieldset from 'components/forms/form-fieldset';
 import FormTextInput from 'components/forms/form-text-input';
 import LoggedOutForm from 'components/logged-out-form';
 import Notice from 'components/notice';
 import { localize } from 'i18n-calypso';
-
-//@TODO move this to wp-login compoenent
 import { getCurrentUser } from 'state/current-user/selectors';
 
 class RequestLoginEmailForm extends React.Component {
+	state = {};
```

Comment:
@stephanethomas -- it was to prevent a reference error in the render method before any state was set. I went ahead and took it out & put a guard in place in ff01d10 which should be even more robust.